src/procedures/procedures.11tydata.yaml

```
@@ -286,6 +286,189 @@ procedureMigrateZigbeeZHAOnly:
       image: z2m-backup-restore-05.png
       content: |
         <ul>
-        <li class="info">All migrated devices have a new entity ID. This means you will need to manually reconfigure automations and dashboard cards.</li>
+        <li class="info">All migrated devices have new entity IDs. This means you will need to manually reconfigure automations and dashboard cards.</li>
           <li class="info">The names that you manually assigned to your devices won't have been migrated. You will have to rename the devices again.</li>
         </ul>
+procedureMigrateConBee:
+  title: Migrate a deCONZ network to SkyConnect using ZHA
+  steps:
+    - title: Update the ConBee stick
+      image: 
+      content: |
+        <ul>
+          <li>If you are using a ConBee stick, it is recommended to have it running firmware version 26720700.</li>
+          <li>If it is not running this version, follow the procedure <b>Update the ConBee stick</b> at the end of this page.</li>
```

```suggestion
          <li>If it is not running this version, follow the procedure <b>Update the ConBee stick</b> at <a href="#update-the-conbee-stick">the end of this page.</a></li>
```

On second thought, I'd guess almost anyone wants to check their firmware version, which is already the first step. Can't we just put this here instead?

---

I had it there first. Then I thought that some people might use a RaspBee?

---

From this FAQ it appears that firmware update works the same way: https://phoscon.de/en/support#faq_hardware

---

@c0ffeeca7 still think it is better if we move this up.

---

you're right. Didn't read carefully. Fixed now. Thank you for the catch.